ci/pipelines/deploy-to-production.yml

```diff
@@ -65,6 +65,13 @@ definitions:
         text: "((.:failure_snippet)) \n
               - <https://cd.gds-reliability.engineering/teams/$BUILD_TEAM_NAME/pipelines/$BUILD_PIPELINE_NAME/jobs/$BUILD_JOB_NAME/builds/$BUILD_NAME|Concourse build #$BUILD_NAME>"
 
+  put_db_migration_slack_notification: &put_db_migration_slack_notification
+    put: slack-notification
+    params:
+      channel: '#govuk-pay-announce'
+      text: ":postgres: FARGATE starting $BUILD_JOB_NAME on production-2\n
+            - <https://cd.gds-reliability.engineering/teams/$BUILD_TEAM_NAME/pipelines/$BUILD_PIPELINE_NAME/jobs/$BUILD_JOB_NAME/builds/$BUILD_NAME|Concourse build #$BUILD_NAME>"
```

Comment:
I take it `BUILD_PIPELINE_NAME`, `BUILD_JOB_NAME`, and `BUILD_NAME` are standard env vars for that running job?

Reply:
Correct ✔️